docs/assets/images/Dockstore-Documentation-horizontal-white.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It doesn't look like you're using this file. If you aren't, can you please delete it.